qa/pull-tester/rpc-tests.sh

```diff
@@ -34,6 +34,7 @@ testScripts=(
     'maxblocksinflight.py'
     'invalidblockrequest.py'
     'rawtransactions.py'
+    'retarget-overflow.py'
```

Comment:
This test should be disabled in `rpc-tests.sh` because it mines serval hundred blocks and it's therefore not ideal for travis/CI (timeouts,etc.).